src/js/providers/html5.js

```diff
@@ -446,6 +446,13 @@ define([
             return currentQuality;
         }
 
+        function _play() {
+            var promise = _videotag.play();
+            if (promise && promise.catch) {
```

Comment:
Since Chrome 50. Other browsers are on board or getting there. We could actually use this in iOS 10 to determine play() won't actually start playback because a user gesture is required.

We also need a similar catch in the commercial project for shaka and caterpillar.